solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -2736,6 +2736,12 @@ private Set<Runnable> getConfDirListeners(final String confDir) {
     return confDirListeners;
   }
 
+  public boolean hasConfDirectoryListeners(final String confDir) {
```

Comment:
so, this method was added just for the JUnit? do we not have any other way to know if there is a watch from Zookeeper client?